src/main/java/com/hedvig/backoffice/graphql/resolvers/GraphQLQuery.java

```diff
@@ -148,4 +145,20 @@ public String me(DataFetchingEnvironment env) {
   public List<ClaimItem> claimItems(UUID claimId) {
     return itemizerService.getClaimItems(claimId);
   }
+
+  public List<VoucherCampaign> findPartnerCampaigns(CampaignFilter filter) {
+    final var code = filter == null ? null : filter.getCode();
+    final var partnerId = filter == null ? null : filter.getPartnerId();
+    final var activeFrom = filter == null ? null : filter.getActiveFrom();
+    final var activeTo = filter == null ? null : filter.getActiveTo();
+    return productPricingService.searchPartnerCampaigns(code, partnerId, activeFrom, activeTo)
+      .stream().map(partnerCampaignSearchResponse ->
+        VoucherCampaign.Companion.from(partnerCampaignSearchResponse)
+      ).collect(Collectors.toList());
```

Comment:
Imo this would be neat, remove all the variables at the top
```suggestion
    final val campaigns = filter == null ?  productPricingService.searchPartnerCampaigns(null, null, null, null) : productPricingService.searchPartnerCampaigns(filter.getCode(), filter.getPartnerId(), filter.getActiveFrom(), filter.getActiveTo())
    return campaigns.stream().map(partnerCampaignSearchResponse ->
        VoucherCampaign.Companion.from(partnerCampaignSearchResponse)
      ).collect(Collectors.toList());
```